cl/search/models.py

```diff
@@ -1923,6 +1995,27 @@ class OpinionCluster(models.Model):
         db_index=True,
         default=False,
     )
+    filepath_local = models.FileField(
+        help_text="Path to Harvard json local storage.",
```

Comment:
Let's make this description generalizable. Something more like, "The source file for the entire cluster. This is used by the Harvard data. See also Opinion.filepath_local for..."